compiler/rustc_middle/src/ty/instance.rs

```diff
@@ -408,7 +410,7 @@ impl<'tcx> Instance<'tcx> {
         let args = tcx.erase_regions(args);
         tcx.resolve_instance(tcx.erase_regions(param_env.and((def_id, args))))
     }
-
+    /// Behaves exactly like [`resolve`], but panics on error.
```

Comment:
Can you keep all the empty lines between the comment and previous function.
```suggestion

    /// Behaves exactly like [`resolve`], but panics on error.
```